MilitaryManager.Attachments.API/Services/StoreService/StoreService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Http;
+
```

Comment:
Move this file to Core or Infrastructure. Meybe it`s better because this logic for work with global services. Need discuss